build.assets/tooling/cmd/build-apt-repos/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"container/list"
+	"fmt"
+	"io/fs"
+	"path/filepath"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/seqsense/s3sync"
+)
+
+func main() {
+	// artifactPath := "/artifacts"
+	// bucketName := "aptly-test20220223185137234100000001"
+	// localBucketPath := "/bucket"
+	// majorVersion := "vTODO"
+	// releaseChannel := "stable"
+	supportedOSs := map[string][]string{
+		"debian": {
+			"stretch",
+			"buster",
+			"bullseye",
+			"bookwork",
+			"trixie",
+		},
+		"ubuntu": {
+			"xenial",
+			"yakkety",
+			"zesty",
+			"artful",
+			"bionic",
+			"cosmic",
+			"disco",
+			"eoan",
+			"focal",
+			"groovy",
+		},
+	}
+
+	args := parseFlags()
+
+	awsSession := session.Must(session.NewSession())
+	aptly := SetupAptly()
+
+	downloadExistingRepo(awsSession, *args.bucketName, *args.localBucketPath)
```

Comment:
Per our external conversations, this syncs the entire repo(s) down. I updated dronegen to store this on a PVC so that we're not pulling the whole thing down every time, only changes.